api/src/main/java/io/minio/MinioClient.java

```diff
@@ -5717,6 +5719,7 @@ public void removeIncompleteUpload(String bucketName, String objectName)
    *
    */
 
+  @Deprecated
```

Comment:
do not make changes in deprecated methods instead you remove them in separate PR